websites/conftest.py

```diff
@@ -55,6 +56,15 @@ def global_admin_user():
     return global_admin_user
 
 
+@pytest.fixture()
+def website_collection_admin():
+    """returns a user with permissions to create / edit Collectins"""
+    user = UserFactory.create()
+    assign_perm(constants.PERMISSION_CREATE_COLLECTION, user)
+    assign_perm(constants.PERMISSION_EDIT_COLLECTION, user)
```

Comment:
Has Ferdi or Peter mentioned who should be able to create/edit collections?  If global admins and/or authors, then there is existing code assigning permissions to those, and these permissions should be added.  If individual users on a case-by-case basis, it might be useful to have a management command for adding permissions to those users.

Reply:
I think for now maybe the easiest thing is to have separate permissions for these?

I updated the permissions to allow users with the EDIT_COLLECTION permission to delete `WebsiteCollectionItem` records as well.

Reply:
I was actually wondering if the collection permissions (add/edit) should be added to the global `Group`s  (global admin, global author) created in `websites.permissions.create_global_groups`: https://github.com/mitodl/ocw-studio/blob/master/websites/permissions.py#L103, or if a separate group should be created for these permissions?

Reply:
I'm really not sure what is the normal / best practice for that, and tbh after reading through all the code in `website/permissions.py` I don't really understand how it works or how I would go about adding a new group/permission/etc

Would I need to create a new group, and then make changes to these functions to create those groups or something? Right now I am using these built-in permissions which (afaict) django automatically creates when you create a new model: https://docs.djangoproject.com/en/3.2/topics/auth/default/#default-permissions

I think maybe the gap in my understanding is seeing what we're doing with the custom groups and whatnot that is different? I don't know if my question even makes sense though, haha

Reply:
Something I hadn't thought of yet, but depends on what management wants - when a particular collection is created, should the admin/owner be able to give other people the ability to edit that particular collection? Similar to websites (the 'Collaborators' link for each site).  If yes, then each collection will need its own edit group (see `websites.permissions. setup_website_groups_permissions` for how that's done for websites).  If not, and this is something only for global admins and authors, and if they should be able to edit all collections, then you don't need to worry about that.

Reply:
My sense is that we'll want something like that in the future, but this relatively simple approach is going to be fine for now.